webapi/Controllers/ServiceOptionsController.cs

```diff
@@ -39,15 +41,24 @@ public class ServiceOptionsController : ControllerBase
     [ProducesResponseType(StatusCodes.Status200OK)]
     public IActionResult GetServiceOptions()
     {
-        return this.Ok(
-            new ServiceOptionsResponse()
+        var response = new ServiceOptionsResponse()
+        {
+            MemoryStore = new MemoryStoreOptionResponse()
             {
-                MemoryStore = new MemoryStoreOptionResponse()
-                {
-                    Types = Enum.GetNames(typeof(MemoryStoreOptions.MemoryStoreType)),
-                    SelectedType = this._memoryStoreOptions.Type.ToString()
-                }
-            }
-        );
+                Types = Enum.GetNames(typeof(MemoryStoreOptions.MemoryStoreType)),
+                SelectedType = this._memoryStoreOptions.Type.ToString()
+            },
+            Version = GetAssemblyFileVersion()
+        };
+
+        return this.Ok(response);
+    }
+
+    private static string GetAssemblyFileVersion()
+    {
+        Assembly assembly = Assembly.GetExecutingAssembly();
+        FileVersionInfo fileVersion = FileVersionInfo.GetVersionInfo(assembly.Location);
```

Comment:
does this refer to a different type of version that what is passed in during deployment?

Reply:
No.  This extracts the version that was set (or not) during build / deployment